cypress/integration/ProcessorTest.ts

```diff
@@ -225,50 +240,50 @@ onlyOn(isEnvironmentType(EnvType.Mocked), () => {
         );
       });
 
-      onlyOn(bugPresent, () => {
-        it("Add filter row", () => {
-          filters[1] = ["data.surname", "String equals", "White"];
+      it("Add filter row", () => {
+        filters[1].Key = "data.surname";
```

Comment:
@jomarko there could be a problem

Reply:
Do you think that there could be: `filters[1]["Key"]`?

Reply:
similar question as above

Reply:
@jomarko  I mean that `.Key` might caused that problem. I am not TS expert too but based on my Java and JS experience:
1.`.Key` reference the property of an object which is saved in the `fitlers` array under index `1`
2. `["Key"]` reference the item in an array/map structure which key is `"Key"` this array/map is saved in the `filters` array under index `1`.

Reply:
@jomarko It seems that problems is still here. Do you think that this could help: `filters.push({ Key: "data.name", Type: "String equals", Value: "John" });`?

Reply:
> @jomarko It seems that problems is still here. Do you think that this could help: `filters.push({ Key: "data.name", Type: "String equals", Value: "John" });`?

The failure was moved. The assertation of the new filter row failed >>> "undefined".

Reply:
@sjamboro done in a separate commit